.npmrc

```diff
@@ -1 +1,3 @@
+use-node-version=18.4.0
```

Comment:
Since it's a _Template_ project, I was thinking using a pinned version instead of a range might not be the best thing to do. However, I guess it's ok for now !

btw : I only guessed your node setup to use the latest stable node version, edit if needed

Reply:
We will start with a fix a see if we want to be more flexible